### PR TITLE
fix zcat bug by using cat for all concatenation

### PR DESCRIPTION
**Pull request by wshands**
fix zcat bug by using cat for all concatenation

### CODE REVIEW

File: docker/wrapper.py [hunk 1]
```diff
@@ -94,29 +94,12 @@ def generate_manifest(sample_tars, sample_singles, sample_pairs, workdir, output
...
     '''
-    Routine to unzip and concatenate input files
-    using zcat which is much faster than using python 
-    gunzip
+    Routine to concatenate input files, gzipped or not using cat
+    For gzipped files this is much faster than using python gunzip
+    see https://www.biostars.org/p/136025/ and https://www.biostars.org/p/81924/
     '''
-    command = 'zcat'
+    command = 'cat'
```
Comment (jvivian) [inline docker/wrapper.py]: Did you test this on more than one sample?  I don't believe you can `cat` files together if they're in gzip format.

Comment (wshands) [inline docker/wrapper.py]: Yes I tried on multiple samples. It does actually work for gzipped files. Take a look at  https://www.biostars.org/p/136025/ and https://www.biostars.org/p/81924/

Comment (jvivian) [inline docker/wrapper.py]: Oh that's great to know, I can refactor some un-DRY code in the process sample step.  Thanks for testing that out.

**Comment by jvivian** [comment]
@wshands — I'm merging master into this branch to copy over the SINGLE-END fix so it's up to date. After tests pass I'll merge.

File: docker/wrapper.py (hunk 1 above)
Comment (wshands) [inline docker/wrapper.py]: We or Treehouse needs to do concordance analysis on the docker output

**Pull request merged**